x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_overview.tsx

```diff
@@ -49,7 +55,11 @@ export const EngineOverview: React.FC = () => {
                 color="text"
               >
                 <EuiFlexGroup alignItems="center">
-                  <EuiIcon size="xxl" type="visTable" color="#98A2B3" />
+                  {hasUnknownIndices ? (
+                    <EuiIcon size="xxl" type="alert" color="#F5BF23" />
```

Comment:
Should we be using `euiTheme.colors` here instead of hash values?

[Eui Colors](https://elastic.github.io/eui/#/theming/colors/values)

Reply:
happy to!